packages/common/src/adapters/track.ts

```diff
@@ -227,35 +232,31 @@ export const stemTrackMetadataFromSDK = (
 
 export const trackMetadataForUploadToSdk = (
   input: TrackMetadataForUpload
-): TrackMetadata => ({
+): TrackFilesMetadata => ({
   ...camelcaseKeys(
     pick(input, [
-      'track_cid',
       'license',
       'isrc',
       'iswc',
-      'genre',
       'is_unlisted',
       'is_premium',
       'premium_conditions',
       'is_stream_gated',
       'stream_conditions',
       'is_download_gated',
-      'orig_file_cid',
       'orig_filename',
       'is_downloadable',
       'is_original_available',
       'bpm',
-      'duration',
       'is_custom_bpm',
       'musical_key',
       'is_custom_musical_key',
       'comments_disabled',
       'ddex_release_ids',
-      'parental_warning_type',
-      'audio_upload_id'
+      'parental_warning_type'
     ])
   ),
+  trackId: OptionalId.parse(input.track_id),
```

Comment:
Is this because we use the same schema for both upload and edit?
I would expect the upload case to not allow passing a trackId since it gets generated.

Reply:
We manually generate the track ids in advance so that we can associate the stem files. So yes upload does optionally accept a track id.

But i would like to split up the upload/edit schema at some point